src/main/java/org/opensrp/repository/postgres/SettingRepositoryImpl.java

```diff
@@ -251,15 +270,15 @@ public SettingsMetadata saveSetting(SettingConfiguration entity, SettingsMetadat
 		}
 		
 		SettingsMetadata settingsMetadata = metadata != null ? metadata : createMetadata(entity, settings.getId());
-		settingsMetadata.setServerVersion(Calendar.getInstance().getTimeInMillis());
+		
 		if (settingsMetadata != null) {
+			settingsMetadata.setServerVersion(Calendar.getInstance().getTimeInMillis());
```

Comment:
Set the `serverVersion` on the `entity`. The `createMetadata` will then extract the server version from  `entity`